### PR TITLE
update Action Group API version for new arm role receiver

### PR DESCRIPTION
**Pull request by RyanD1**

### CODE REVIEW

**Comment by RyanD1** [comment]
@Tiano2017 Hi we have some pending api update could you take a look?

File: schemas/2018-09-01/Microsoft.Insights.json [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+        "functionAppResourceId",
+        "functionName",
+        "httpTriggerUrl"
+      ],
+      "description": "A azure function receiver."
+    },
+    "ArmRoleReceiver": {
+      "type": "object",
+      "properties": {
+			      "name": {
```
Comment (TianoMS) [inline schemas/2018-09-01/Microsoft.Insights.json]: >			 [](start = 0, length = 3)

use spaces.

File: schemas/2018-09-01/Microsoft.Insights.json [hunk 2]
```diff
@@ -0,0 +1,460 @@
...
+        "name",
+        "countryCode",
+        "phoneNumber"
+      ],
+      "description": "A voice receiver."
+    },
+    "LogicAppReceiver": {
+      "type": "object",
+      "properties": {
+			"name": {
```
Comment (TianoMS) [inline schemas/2018-09-01/Microsoft.Insights.json]: >			 [](start = 0, length = 3)

use spaces.

Comment (TianoMS) [reply schemas/2018-09-01/Microsoft.Insights.json]: please format the json file. 

---
In reply to: [215785063](https://github.com/Azure/azure-resource-manager-schemas/pull/502#discussion_r215785063) [](ancestors = 215785063)

**Pull request closed without merging**